web/src/components/molecules/Asset/UploadModal/tab1.tsx

```diff
@@ -0,0 +1,27 @@
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Upload, { UploadProps } from "@reearth-cms/components/atoms/Upload";
+
+const { Dragger } = Upload;
+
+type Props = {
+  uploadProps: UploadProps;
+};
+
+const Tab1: React.FC<Props> = ({ uploadProps }) => {
```

Comment:
Can you think of a more meaningful name than Tab1? Also, are the second and third tabs going to be done later? If so, it would be nice to add that to the PR description.